src/graph/graph.people.ts

```diff
@@ -9,18 +9,58 @@ import { Contact, Person, User } from '@microsoft/microsoft-graph-types';
 import { IGraph } from '../IGraph';
 import { prepScopes } from '../utils/GraphHelpers';
 
+/**
+ * Person Type enum
+ *
+ * @export
+ * @enum {number}
+ */
+export enum PersonType {
+  /**
+   * Any type
+   */
+  Any = 0,
+
+  /**
+   * A Person such as User or Contact
+   */
+  // tslint:disable-next-line:no-bitwise
+  Person = 1 << 0,
+
+  /**
+   * A group
+   */
+  // tslint:disable-next-line:no-bitwise
+  Group = 1 << 1
+}
+
 /**
  * async promise, returns all Graph people who are most relevant contacts to the signed in user.
  *
  * @param {string} query
+ * @param {number} [top=10] - number of people to return
+ * @param {PersonType} [personType=PersonType.Person] - the type of person to search for
  * @returns {(Promise<Person[]>)}
- * @memberof Graph
  */
-export async function findPerson(graph: IGraph, query: string): Promise<Person[]> {
+export async function findPeople(
+  graph: IGraph,
+  query: string,
+  top: number = 10,
+  personType: PersonType = PersonType.Person
+): Promise<Person[]> {
   const scopes = 'people.read';
+
+  let filterQuery = '';
+
+  if (personType !== PersonType.Any) {
+    filterQuery = `personType/class eq '${PersonType[personType]}'`;
```

Comment:
This is relatively unusual. Why are you creating a bit field if you just end up using the string values assigned as the enum names? As constructed, this is very delicate, as it's not obvious that the names are sensitive at all. I would recommend using a string enum or simply even a union of string literals.

Reply:
Good point - I was attempting to follow the same pattern as `GroupType` which does use the individual bits - but in this case it's unnecessary. Switched to string enums

Reply:
You can just use:
```
`personType/class eq '${personType}'`
```

Reply:
Done